site/src/pages/components/link-area-delegation-explainer.mdx

```diff
@@ -0,0 +1,191 @@
+
+## The clikcable container or "card" pattern
+Commonly a clickable container, often referred to as a "card", will present a graphic or other media, text, a primary call to action (link or button) as well as other supplementary interactive elements:
+
+<figure>
+  <img src="https://github.com/user-attachments/assets/27e5076d-bc98-4fb8-a429-e8462ff9024c" 
+       width="400" 
+       alt="Screenshot of a Reddit topic 'card' from the r/cats subreddit. Shows time of publication, r/cats as a hyperlink, a join and more options button, followed by the title of the post and a graphic of a cat pearing out a window while its owner takes its picture. up/down vote, comment, award and share buttons follow the graphic.">
+
+  <figcaption>
+    <p>With this Reddit example, the whole card is clickable and doing so and would navigate a user to the Reddit post. But there are also other interactive elements (up/down vote, comments, join, the sub-reddit link, etc.) are also clickable. These appear to be "nested" within the clickable card, but due to work arounds, this is not the case.</p>
+  </figcaption>
+</figure>
+
+### Existing ways to achieve clickable containers
+Today, making a container/card clickable can be achieved by a few different approaches, each with its own tradeoffs:
+1. Using the `a` element to wrap the content that needs to be clickable.  The issue with this is:
+  a. doesn't support nested interactive elements without creating accessibility issues.
+  b. using an `a` element to wrap content is not supported in certain contexts - e.g., such as trying to wrap a row within a table.  The HTML parser will remove such hyperlinks from the rendered table markup.
+  c. even without nested interactive elements, the accessible name of the hyperlink can become overly verbose, or may not be exposed as all depending on the element the hyperlink wraps.
+2. Using CSS to position / stack interactive elements. For instance, using CSS to put the link, or a link's `::before` or `::after` pseudo element "behind" the "nested" interactive elements, so that the link catches clicks that are not handled by these other interactive targets. The issue here is:
+  a. no standardized way to do this, with some implementations better than others - particularly such implementations that use duplicate links, creating redundancy for keyboard and/or screen reader users (even if the duplicate link is `tabindex=-1`).
+  b. this sort of pattern often makes it difficult if not impossible for some users to select what appears to be static text. 
```

Comment:
```suggestion
  b. this sort of pattern often makes it difficult if not impossible for some users to select what appears to be static text. 
  c. Such a pattern usually relies on all intermediate elements between the container/card and the primary action within to have `display: static`. Depending on the visual design requirements, this may not be feasible.
```